src/copa_game/ports/http_out.clj

```diff
@@ -1,14 +1,17 @@
 (ns copa-game.ports.http-out
-  (:require [copa-game.adapters :as adapters]
+  (:require [parenthesin.components.http :as components.http]
             [copa-game.schemas.types :as schemas.types]
-            [parenthesin.components.http :as components.http]
-            [schema.core :as s]))
+            [schema.core :as s]
+            [copa-game.schemas.out :as schemas.out]))
 
-(s/defn get-teams
-        [http :- schemas.types/HttpComponent]
-        (->> {:url "https://l3-processoseletivo.azurewebsites.net/api/Competidores?copa=games"
-              :as :json
-              :method :get}
-             (components.http/request http)
-             :body
-             adapters/wire->teams))
\ No newline at end of file
+(s/defn get-teams :- [schemas.out/Team]
+  [{:keys [http]} :- schemas.types/Components]
+  (->>
+    (components.http/request http {:url "https://l3-processoseletivo.azurewebsites.net/api/Competidores?copa=games"
+                                   :as :json
+                                   :method :get})
+    :body))
+
+(comment
+(def http (:http @system-atom))
+  (get-teams http))
```

Comment:
There are no problems leaving comments in the code, but I think a description would always be nice? Adding indentation could also improve readability. Also note the atom is in another namespace, so I think we need to fully qualify it.
```clj
(comment
  "Request all the available teams"
  (def http (:http copa-game.server/@system-atom))
  (get-teams {:http http}))
```
(or you can just remove the comment)

Reply:
✂️